pkg/ensurance/collector/nodelocal/cpu.go

```diff
@@ -64,18 +85,31 @@ func collectCPU(nodeState *nodeState) (map[string][]common.TimeSeries, error) {
 	usagePercent := calculateBusy(nodeState.latestCpuState.stat, currentCpuState.stat)
 	usageCore := usagePercent * float64(nodeState.cpuCoreNumbers) * 1000 / types.MaxPercentage
 
-	klog.V(6).Infof("CpuCollector collected,usagePercent %v, usageCore %v", usagePercent, usageCore)
+	cpuSet := nodeLocalContext.exclusiveCPUSet()
```

Comment:
Can we modified it to if the exclusiveCPUSet is not set here , MetricNameExclusiveCPUIdle set to zero directly?

Reply:
Yes, I've defined a DefaultExclusiveCPUSet to return a zero CPUSet, so there should be no need to judge it here. Of course, adding a judgment will make the program more robust. I will fix it.
![image](https://user-images.githubusercontent.com/29157728/160058372-b140ca26-b42b-430f-af3a-a39f5b3b9d7f.png)
![image](https://user-images.githubusercontent.com/29157728/160058844-e13b7885-fea3-46fe-84f9-f3b10653bb5b.png)